apps/dg/english.lproj/function_strings.json

```diff
@@ -1179,12 +1179,12 @@
       {
         "name": "stringToLookIn",
         "type": "string",
-        "description": "The string you want to find."
+        "description": "The string you want to find. Note that the string needs to be inside quotes."
```

Comment:
Suggest:
A string (in quotes) or an expression that evaluates to a string.

This applies to a number of the changes below this one.